conda-forge.yml

```diff
@@ -12,3 +12,5 @@ provider:
   linux_ppc64le: default
 os_version:
   linux_64: cos7
+  linux_aarch64: cos7
+  linux_ppc64le: cos7
```

Comment:
Thanks Axel! 🙏

We already have `cos7` in `os_version`

Reply:
Yeah, I saw that this pattern got used widely in the cuda-related feedstocks. I'm not the biggest fan because it's a bit convoluted how the `os_version` ends up influencing the `c_stdlib_version` - which is also why I'd prefer to have this in CBC (or at least, in addition to setting `os_version: cos7`), not least because it's explicit and is the canonical definition for what populates `{{ stdlib("c") }}`.

I mean, up to you as maintainers if you think this works well enough, but thought I'd mention it.